roles/cifmw_cephadm/templates/ceph.conf.j2

```diff
@@ -14,7 +14,7 @@ fsid = {{ cifmw_cephadm_fsid }}
 mon host = {{ cifmw_cephadm_first_mon_ip }}
 
 {# Build CephConfigOverrides #}
-{% for key, value in (ceph_conf_overrides | default({})).items() %}
+{% for key, value in (cifmw_ceph_rgw_config | default({})).items() %}
```

Comment:
Since this template will only contain RGW configuration (and we don't need to support `ceph_conf_overrides`) perhaps it should be renamed to `ceph_rgw.conf.j2` so as to avoid confusion.

You could then remove the call in pre.yml to create this template and call it instead from rgw.yml since it's not needed until RGW configuration and not used at bootstrap. That would make the code more clear.

Reply:
@fultonj Thanks and agree with you. As overrides are only related to rgw, i will make sure to move/add the code in rgw.yaml.

Reply:
If we're going to create a template to render RGW overrides, I'm wondering if it makes sense to you to have a generic conf file where we not only render RGW, but we leave space to any potential override we want to inject in the ceph-cluster (it would be useful for job override purposes).
In that case the struct should be extended to have multiple entries:

```yaml
ceph_conf_overrides:
    cifmw_ceph_rgw_config:
       <params>
    other_overrides:
       <params>
```

In both cases (the scope is reduced to RGW or not), I agree to move it to a `ceph_<something>.conf.j2` to avoid confusion and for better maintenance.

- <something> can be `overrides` or `rgw`, I really do not want to enforce a particular scope here; it depends if we want an interface to `assimilate` more than just RGW, or not, and if in the future we plan for this.

Reply:
As discussed , template file is renamed as `ceph_rgw.conf.j2`, and it rendered and applied in rgw.yaml